homeassistant/components/conversation/__init__.py

```diff
@@ -101,16 +100,14 @@ async def websocket_process(
     msg: dict[str, Any],
 ) -> None:
     """Process text."""
-    connection.send_result(
-        msg["id"],
-        await _async_converse(
-            hass,
-            msg["text"],
-            msg.get("conversation_id"),
-            connection.context(msg),
-            msg.get("language"),
-        ),
+    result = await _async_converse(
+        hass,
+        msg["text"],
+        msg.get("conversation_id"),
+        connection.context(msg),
+        msg.get("language"),
     )
+    connection.send_result(msg["id"], result.as_dict())
```

Comment:
this is a breaking change for the frontend and will need a PR to fix it in frontend.

Reply:
https://github.com/home-assistant/frontend/pull/14763

Reply:
https://github.com/home-assistant/frontend/pull/14763